source/tutorials/importing.rst

```diff
@@ -84,12 +85,14 @@ Format description
 
 Paired-end "`Earth Microbiome Project (EMP) protocol`_" formatted reads should have three ``fastq.gz`` files total:
```

Comment:
Can you also update the Single-end EMP stuff above, similarly, since it too is a directory format, and not a single-file format.

Reply:
Please apply the same edits you have made in these sections on paired-end EMP to the single-end EMP section, above.